src/lib/config.ts

```diff
@@ -1,10 +1,38 @@
 import * as _ from 'lodash';
+import * as memoizee from 'memoizee';
+
+const openVpnConfig = `
+client
+remote VPN_DETAILS
```

Comment:
Can you please make it more apparent that this is a placeholder that will be replaced later on? Maybe wrap it in curly braces or percent signs (`{{ VPN_DETAILS }}` or `%%VPN_DETAILS%%`)?